apps/src/lib/levelbuilder/shapes.jsx

```diff
@@ -64,6 +69,22 @@ export const activityShape = PropTypes.shape({
   activitySections: PropTypes.arrayOf(activitySectionShape)
 });
 
+export const levelShapeForScript = PropTypes.shape({
+  position: PropTypes.number,
+  activeId: PropTypes.number,
+  ids: PropTypes.arrayOf(PropTypes.number),
+  kind: PropTypes.string,
+  skin: PropTypes.string,
+  videoKey: PropTypes.string,
+  concepts: PropTypes.string,
+  conceptDifficulty: PropTypes.string,
+  progression: PropTypes.bool,
+  named: PropTypes.bool,
+  bonus: PropTypes.bool,
+  assessment: PropTypes.bool,
+  challenge: PropTypes.bool
+});
+
 export const lessonShape = PropTypes.shape({
```

Comment:
is this shape only used for script editing? If so it could be helpful to call this `lessonShapeForScript` too.

Reply:
Well I think the levelShapeForScript is a temporary thing. I didn't want to tackle updating the level shape for script edit in this PR on top of everything else. I think we should probably move the script edit page over to the same levelShape as the lesson edit page because then we can support things like variants.

Reply:
Tracking: https://codedotorg.atlassian.net/browse/PLAT-389

Reply:
I am not sure I believe that the shapes should be the same for script edit page and lesson edit page. there is a relatively small amount of info we need to show a lesson on the script edit page. so it could make sense for the shapes to be separate. in a way this could relate to the comment that @jamescodeorg made in Slack -- a single levelShape and lessonShape make sense if we're using the DB representation of the object. but if we're just trying to capture the minimum info needed by each feature then separate could make more sense.

Reply:
either way this looks like a good stopping point for this PR. let's discuss before proceeding on the Jira item you linked to.